rebound/librebound.py

```diff
@@ -64,6 +64,14 @@ def additional_forces(self,func):
             # C function pointer
             self.clibrebound.set_additional_forces_with_parameters(func)
 
+    @property
+    def post_timestep_modifications(self):
+        return self.fp   # might not be needed
```

Comment:
This should not be `self.fp` (which is the function pointer for `additional_forces`.